x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -36,9 +42,14 @@ type rttMonitor struct {
 	cancelFn      context.CancelFunc
 }
 
-func newRttMonitor(cfg *rttConfig) *rttMonitor {
+func newRTTMonitor(cfg *rttConfig) *rttMonitor {
 	ctx, cancel := context.WithCancel(context.Background())
+	// Determine the number of samples we need to keep to store the minWindow of RTT durations. The
+	// number of samples must be between [5, 500].
+	numSamples := int(math.Max(minSamples, math.Min(maxSamples, float64((cfg.minRTTWindow)/cfg.interval))))
```

Comment:
The previous behavior for `cfg.interval = 0` is that `rttMonitor.start()` would panic because calling `time.NewTicker` with 0 panics. The new behavior is that `newRTTMonitor()` will panic because it's trying to divide by 0.

Do you think adding an explicit check and panic to clarify that 0 isn't a supported value is a good idea?